packages/offix-client/src/OfflineClient.ts

```diff
@@ -60,13 +61,12 @@ export class OfflineClient implements ListenerProvider {
   /**
   * Initialize client
   */
-  public async init(): Promise<ApolloOfflineClient> {
+  public async init(httpOptions?: HttpLink.Options): Promise<ApolloOfflineClient> {
```

Comment:
No strong opinion but:
I would prefer to go with terminating link for couple reasons:

- It is just easier to migrate to offix by passing your already existing link to init rather than figuring out how to replicate those changes to offix.
- Using terminating link allows people to configure the flow with not only Http
- Using HttpLink.Options make a strong connection on Apollo lib version

The drawback of the terminating link is that some of the config parameters will be ignored.